.gitignore

```diff
@@ -128,6 +128,9 @@ eng/tools/**/dist
 !eng/tools/**/cmd/*.js
 !eng/tools/**/eslint.config.js
 
+# TypeScript cache
+*.tsbuildinfo
```

Comment:
we change this one to be in a `temp/` dir in typespecs repo so you can have a `clean` script that can be used to delete `dist/` `temp/`.
It happens that the cache sometimes is out of date doesn't want to refresh.

Just a FYI, not asking for a change